src/test/java/com/wikia/webdriver/elements/mercury/components/Header.java

```diff
@@ -0,0 +1,57 @@
+package com.wikia.webdriver.elements.mercury.components;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class Header extends BasePageObject {
+
+  @FindBy(css = ".ember-application .wiki-page-header")
+  private WebElement header;
+
+  @FindBy(css = ".ember-application .wiki-page-header__title")
+  private WebElement pageTitle;
+
+  public boolean isHeaderVisible() {
+    try {
+      return header.isDisplayed();
+    } catch (TimeoutException | ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPageTitleVisible() {
+    try {
+      return pageTitle.isDisplayed();
+    } catch (TimeoutException | ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isHeroImageVisible() {
+    try {
+      return header.getAttribute("class").contains("has-hero-image");
+    } catch (TimeoutException | ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public String getHeroImageCssValue(String property) {
+    try {
+      return header.getCssValue(property);
+    } catch (TimeoutException | ElementNotFoundException e) {
+      return "";
+    }
+  }
+
+  public boolean isHeroImageSquare() {
+    try {
+      return header.getSize().getHeight() == header.getSize().getWidth();
+    } catch (TimeoutException | ElementNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)